src/ADAL.Common/OAuth2Response.cs

```diff
@@ -169,6 +173,11 @@ public static AuthenticationResult ParseTokenResponse(TokenResponse tokenRespons
             }
             else if (tokenResponse.Error != null)
             {
+                if (!String.IsNullOrEmpty(tokenResponse.Claims) && tokenResponse.Error == AdalErrorMessage.InteractionRequired)
```

Comment:
Is this error message or error code that we're comparing here? Should probably only be comparing the codes, not the error messages (which might be subject to change over time, or by language).